src/main/java/com/iexec/worker/docker/ComputeMeta.java

```diff
@@ -0,0 +1,22 @@
+package com.iexec.worker.docker;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/* /!\ this probably should be split */
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ComputeMeta {
+    
+    private String chainTaskId;
+    private boolean isSuccessfulCompute;
```

Comment:
If we keep it we can rename fields to
isPreComputed
isComputed
isPostComputed

Reply:
Done.